client/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Details Module</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="text/javascript" src="bundle.js"></script>
```

Comment:
I remember HR saying in their style guide to not name script type text/js. check style guide on learn